src/views/index.pug

```diff
@@ -7,9 +7,9 @@ block content
                 .dynamic-border
                 .card__preview__front--first
                     .card__preview__front-chip
-                        +addPicture('chip.png', 'chip-image', 'display: block;max-width:50px;max-height:40px;')
```

Comment:
using an inline style is bad practice, use a predefined CSS class name, e.g. `.chip-image`